Uni_life_tracker/app/src/main/res/layout/activity_expenses_landing.xml

```diff
@@ -0,0 +1,132 @@
+                android:layout_height="wrap_content"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:errorEnabled="true">
+        </com.google.android.material.textfield.TextInputLayout>
+
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="240dp"
+                android:layout_marginStart="95dp"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_amount"
+                android:inputType="text"
+        />
+        <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/e_add_date"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:errorEnabled="true">
+        </com.google.android.material.textfield.TextInputLayout>
+
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="240dp"
+                android:layout_marginStart="95dp"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_date"
+                android:inputType="text"
+        />
+        <ImageButton
+                android:id="@+id/plus_button"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                android:layout_width="59dp"
+                android:layout_height="65dp"
+                android:src="@drawable/plus_button_round"
+                android:padding="12dp"
+                android:scaleType="fitCenter"
+                android:background="#0000ffff"
+                app:layout_constraintHorizontal_bias="0.049"
+                app:layout_constraintVertical_bias="0.0"
+                app:layout_constraintStart_toStartOf="parent"
+                android:layout_marginStart="180dp"
+                android:contentDescription="@string/left_arrow"/>
+        <Button
+                android:layout_width="300dp"
+                android:layout_height="80dp"
+                android:layout_marginTop="2dp"
+                android:layout_marginStart="65dp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                android:id="@+id/e_go_to_view_button"
+                android:text="@string/go_to_expenses_view"
+                android:backgroundTint="#070707"/>
+
```

Comment:
I think this button should check if all the inputs have been filled up, similar to what I did in the signup page where there are error messages if it's not filled. You can reference both my signup page XML and Java to look into how I did it, just so the user doesn't leave it blank

Reply:
sounds good